data-access/seedwork/event-bus-seedwork-node/sync-domain-event-bus.ts

```diff
@@ -0,0 +1,51 @@
+//// Sync Domain Event
+export interface SyncDomainEventPayloadBaseType {}
```

Comment:
Your suggestion is good, but we consider events as well-defined objects (although, currently, they have a very lean definition). So we prefer to implement class definitions for events instead of string  enums